scripts/workflows/wait_for_required_workflows.js

```diff
@@ -57,7 +57,7 @@ module.exports = async ({github, context}) => {
     while (now <= deadline) {
         const checkRuns = await github.paginate(github.rest.checks.listForRef, {
             owner: 'cloudquery',
-            repo: 'cloudquery',
+            repo: context.payload.repository.name,
```

Comment:
```suggestion
            repo: context.repo.repo,
```

See https://github.com/actions/github-script/tree/6f00a0b667f9463337970371ccda9072ee86fb27#comment-on-an-issue.

I think this works for all types of payloads